ranger/data/scope.sh

```diff
@@ -290,6 +292,45 @@ handle_mime() {
             xls2csv -- "${FILE_PATH}" && exit 5
             exit 1;;
 
+        ## SQLite
+        *sqlite3)
+            ## Preview as text conversion
+            local tables=""
+            [ "$SQLITE_TABLE_LIMIT" -ge 0 ] || SQLITE_TABLE_LIMIT=20  # fallback to 20 if an invalid value is given
```

Comment:
I prefer letting the tool report erroneous arguments/configuration otherwise users end up confused why their setting doesn't seem to be respected.

Reply:
Whether I should show report erroneous to `stdin` and `exit 5`, or simply `exit 1` (no preview).

Reply:
The `&&` condition will fail so everything'll probably fall through to the `exit 1` at the end. This is not something that requires special handling iirc.